config/application.rb

```diff
@@ -23,5 +23,9 @@ class Application < Rails::Application
     config.middleware.use BounceBrowserconfig
 
     config.mount_javascript_test_routes = false
+
+    config.action_view.field_error_proc = proc { |html_tag, _|
```

Comment:
I think this might be worth breaking into it's own commit to help explain what it does and why the change was made?